tetris_description/urdf/tetris.gazebo.xacro

```diff
@@ -16,7 +16,7 @@
       <rightJoint>left_wheel_joint</rightJoint>
       <wheelSeparation>${wheel_separation}</wheelSeparation>
       <wheelDiameter>${wheel_diameter}</wheelDiameter>
-      <torque>5</torque>
+      <torque>0.8</torque>
```

Comment:
I put acceleration limit, to prevent the flipover motion of the robot, due to sudden stop/start.